tasks/scripts/containerd-integration

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+set -e -u
+
+export GOPATH=$PWD/gopath
+export PATH=$GOPATH/bin:$PATH
+
+source ci/tasks/scripts/docker-helpers.sh
```

Comment:
hmmm, this file's name becomes misleading with this PR. If the same helper is used for containerd, it's not so much about docker so much as... allowing nested containers? honestly that `sanitize_cgroups` function has always gone way over my head.

Reply:
> this file's name becomes misleading with this PR. [...] it's not so much about docker so much as... allowing nested containers?

oh totally! that's the sole purpose for `source`ing that file in

do you think it'd be better having a "cgroups-helpers.sh" (or a better name) that we consume in both `docker-helpers` & `containerd-integration`?

Reply:
Yeah, that sounds ideal! Also I'd just like to better understand what that function does haha

Reply:
>  I'd just like to better understand what that function does haha

I just added some comments to make it clearer 😁  

see f14a660928e3624100ac65449869a83265b1ab38

Reply:
btw, I went with the "singular form" (`cgroup`) rather than plural (`cgroups`):

>  "cgroup" stands for "control group" and is never capitalized.  **The
singular form is used to designate the whole feature and also as a
qualifier as in "cgroup controllers"**.  When explicitly referring to
multiple individual control groups, the plural form "cgroups" is used.

Reply:
That's awesome, thanks!